### DISCUSSION TITLE
What span of time does a "Frame" represent in the acoustic features?

### DISCUSSION THREAD

**Discussion by akre96** [bklynhlth/openwillis#146] [q&a for the user community] [answered]
I have a 32kHz recorded .wav audio file and I'm seeing roughly 100 frames per second. Will OpenWillis always report exactly 100 frames per second for the acoustic features? Can this be set?

**Comment by GeorgeEfstathiadis** [comment, marked as answer]
Yes that's right. The framewise output of the acoustics function is determined by a frame window of 10ms or (100 frames per second). This is independent of the audio file's fps. In the current version this cannot be changed.

is there a specific reason you’re considering a different frame window for extracting acoustic features? Do you believe a finer or coarser time resolution would be meaningful for your analysis?

**Reply by akre96** [reply]
No specific reason for a different frame rate. I just wanted to confirm that mapping to make sure I'm aligning to other data sources correctly. 
It may be useful as a feature to return a column for timestamp of beginning of frame, but that's not a hard conversion to do myself afterwards.

Thanks!
